dist/s_define.list

```diff
@@ -22,6 +22,8 @@ WT_CACHE_LINE_PAD_END
 WT_CONN_CHECK_PANIC
 WT_DEADLOCK
 WT_DEBUG_BYTE
+WT_DIAG_SINGLE_THREAD_CHECK_START
+WT_DIAG_SINGLE_THREAD_CHECK_STOP
```

Comment:
Minor nit - this is a really long name and `DIAG` and `CHECK` are somewhat redundant in this context.